### PR TITLE
Adding title to history

### PR DESCRIPTION
**Pull request by zafarshafaq**

### CODE REVIEW

File: asims_server/src/main/java/com/nsia/service/impl/InstanceServiceImpl.java [hunk 1]
```diff
@@ -81,9 +84,14 @@
...
     private EntityManager entityManager;
 
+    @Autowired
+    private InstanceHistoryService instanceHistoryService;
+
     @Autowired
     private ReportService reportService;
 
+    @Autowired
+    private InstanceWatcherServiceImpl iwsi;
```
Comment (hpardess) [inline asims_server/src/main/java/com/nsia/service/impl/InstanceServiceImpl.java]: Please follow the naming convention like instanceWatcherService... Also the variable type should be interface not implementation class...
Private InstanceWatcherService instanceWatcherService;

File: asims_server/src/main/resources/application-test.properties [hunk 2]
```diff
@@ -34,4 +34,4 @@ spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.properties.hibernate.hbm2dml.import_files=/data/test-data.sql
 
 # Email
-spring.mail.to=hameedullah.pardess@gmail.com
\ No newline at end of file
+spring.mail.to=zafarshafaq34@gmail.com
```
Comment (hpardess) [inline asims_server/src/main/resources/application-test.properties]: Please remove your email from commit.

File: asims_server/src/main/java/com/nsia/service/impl/InstanceServiceImpl.java (hunk 1 above)
Comment (zafarshafaq) [reply asims_server/src/main/java/com/nsia/service/impl/InstanceServiceImpl.java]: Dear Pardess, 
If I move the (Send Email Function) from implementation to interface. I should implement all interface functions (Everywhere I use it). Should I do it?

Comment (zafarshafaq) [reply asims_server/src/main/java/com/nsia/service/impl/InstanceServiceImpl.java]: Dear Pardess I observed the naming convention. Please share your comment about moving sendEmail function from implementation to interface?

File: asims_server/src/main/resources/application-test.properties (hunk 2 above)
Comment (zafarshafaq) [reply asims_server/src/main/resources/application-test.properties]: removed.

File: asims_server/src/main/java/com/nsia/service/impl/InstanceServiceImpl.java (hunk 1 above)
Comment (hpardess) [reply asims_server/src/main/java/com/nsia/service/impl/InstanceServiceImpl.java]: you should not use InstanceWatcherServiceImpl but use the interface like InstanceWatcherService

**Pull request merged**